icons/plot.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 12c0-.4-.3-.8-.8-1.1l-7.6-4.5a2.93 2.93 0 0 0-3.1 0l-7.6 4.5c-.6.3-.9.7-.9 1.1s.3.8.8 1.1l7.6 4.5a3.3 3.3 0 0 0 3 0l7.6-4.5c.7-.3 1-.7 1-1.1" />
+  <path d="m17.5 8.7-11 6.6" />
+  <path d="m6.6 8.8 10.8 6.4" />
```

Comment:
Is the use case a plot of land, then maybe something like this could be added.
![icon (59)](https://github.com/lucide-icons/lucide/assets/25524993/ade8f70e-cbef-4ccf-aa70-4075c8cb6dc7)

Naming it just plot is kinda unclear, maybe something like `land-parcel` is better in that case.